Unix/installbuilder/datafiles/Linux.data

```diff
@@ -61,6 +61,9 @@ ResolveSystemdPaths()
 }
 
 RemoveGenericService() {
+    if [ "$OMI_DISABLE_SERVICE_START" ]; then
+        return 1
```

Comment:
`return 0` here to be safe. We don't want a non-zero exit code "leaking out" to calling script.